1001-2000/LIT1295Dersan.xml

```diff
@@ -138,413 +146,1482 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                  <label xml:lang="en">Homily Dǝrātewos and Tewobǝstǝyā</label>
+                  <note>Ascribed to <persName ref="PRS10425Yohanne"/> in <ref type="mss" corresp="BMLacq776"/>.</note>
+                  <div type="textpart" subtype="incipit">
+                  <ab>
+                     <l n="1">፩፤ ስምዑ፡ ዘንተ፡ ኵልክሙ፡ አሕዛብ፡ ወአጽምዑ፡ ኵልክሙ፡ እለ፡ ትነብሩ፡ ውስተ፡ ዓለም፡ ዕድ፡ ወአንስት፡ እለ፡ ሀሎክሙ፡ ውስተ፡ ዝንቱ፡ ምስጢር፡ ከመ፡ 
+                        ታእምሩ፡ ኀይለ፡ ወተአምረ፡ ዘገብረ፡ እግዚአብሔር፡ ከመ፡ ያርኢ፡ ዕበዮ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ከመ፡ ይስአል፡ ኵሎ፡ ጊዜ፡ ወከመ፡ ይትወከፍ፡ ኵሎ፡ ሰብአ።</l>
+                     <l n="2">፪፤ ይስእል፡ ኀበ፡ እግዚአብሔር፡ ለእለ፡ ይትአመኑ፡ በጸሎቱ፡ እስመ፡ ይውህቦሙ፡ ኀይለ፡ ወክብረ፡ በከመ፡ ምግባሮሙ።</l>
+                     <l n="3">፫፤ ወሀሎ፡ አሐደ፡ ብእሲ፡ ዘርቱዕ፡ ሃይማኖቱ፡ ዘስሙ፡ <persName ref="PRS11306Deratewos">ዱራታኦስ፡</persName> ወስመ፡ ብእሲቱ፡ 
+                        <persName ref="PRS11307Tewobesteya">ቴዎብስትያ።</persName></l>
+                     <l n="4">፬፤ ወይነብሩ፡ ውስተ፡ አሐቲ፡ ሀገር፡ እንተ፡ ስማ፡ <placeName ref="LOC6571Qalonya">ቆሎንያ፡</placeName> ወቦሙ፡ ብዙኅ፡ ንዋየ፡ 
+                        ወርቅ፡ ወብሩር፡ አልህምት፡ ወአባግዕ፡ አዕዱግ፡ ወአግማል፡ አብቅልት፡ ወአፍራስ።</l>
+                     <l n="5">፭፤ ወይምሕሩ፡ ነዲያነ፡ ወምስኪናነ፡ <persName ref="PRS11306Deratewos">ዱራታኦስ፡</persName> ወብእሲቱ፡ 
+                        <persName ref="PRS11307Tewobesteya">ቴዎብስትያ፡</persName> ወይገብሩ፡ ብዙኀ፡ ምጽዋተ፡ ለለ፡ ኵሉ፡ አውራኅ፡ አመ፡ ዐሡሩ፡ ወሠኑዩ፡ ዕለት፡
+                        በበዓለ፡ ሚካኤል፡ ሊቀ፡ መላእክት።</l>
+                  </ab>
+                  </div>
+                  <div type="textpart" subtype="explicit">
+                     <ab>
+                        <l n="67">፷፯፤ <persName ref="PRS11306Deratewos">ዱራታኦስ፡</persName> 
+                           ወ<persName ref="PRS11307Tewobesteya">ቴዎብስትያ፡</persName> አፈድፋዱ፡ ገቢረ፡ ሠናይ፡ እምቀዳሚ፡ ግብሮሙ፡ ወፈጸሙ፡ ገድሎሙ፡ 
+                           እስከ፡ ይበጽሕ፡ ዕለተ፡ ዕረፍቶሙ።</l>
+                        <l n="68">፰፰፤ ወኵሎሙ፡ ሕዝብ፡ አምኑ፡ በጸሎቶሙ፡ ወበቃለ፡  ዚአሆሙ፡ በእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ዘሎቱ፡ ስብሐት፡ ምስለ፡ ኄር፡ አቡሁ፡ ወመንፈስ፡ ቅዱስ፡
+                           ማሕየዊ፡ እምይእዜ፡ ወእስከ፡ ለዓለመ፡ ዓለም፡ አሜን።</l>
+                        <l n="69">፷፱፤ በረከቱ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ወረድኤቶሙ፡ ለ<persName ref="PRS11306Deratewos">ዱራታኦስ፡</persName>
+                           ወለ<persName ref="PRS11307Tewobesteya">ቴዎብስትያ፡</persName> ይኩኖ፡ ሠራዬ፡ ኀጢአት፡ ለገብረ፡ እግዚአብሔር፡ ዕገሌ፡ ለዓለመ፡ ዓለም፡
+                           አሜን።</l>
+                     </ab>
+                  </div>
+               </div>        
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="MiracleTreasure">
+                  <label xml:lang="en">Miracle on the poor peasant who found a treasure in a fish</label>
+                  <note>This miracle is transmitted in the section for Gǝnbot in <ref type="mss" corresp="BAVet121"/>.</note>
+                  <div type="textpart" subtype="incipit">
+                  <ab>
+                     <l n="70">፸፤ ተአምሪሁ፡ ለቅዱስ፡ ሚካኤል፡ መልአከ፡ ኪዳኑ፡ ለክርስቶስ፡ ንጉሠ፡ ስብሐት፡ ትንብልና፡ ስእለቱ፡ ወአስተብቍዖቱ፡ ያድኅነነ፡ እምዕለት፡ እኪት፡ 
```

Comment:
if this is a new unit/chapter - then why continue line numbering? each sub-unit starts with 1 I think

Reply:
The numbering is according to the volume of the EOTC edition. The text in this edition is not divided in homily, miracle, commemorative note and salam as in our record.

Reply:
I will change the numbering and try to express the difference in a note.